packages/ckeditor5-ui/docs/_snippets/examples/bootstrap-ui-inner.js

```diff
@@ -305,7 +305,12 @@ BootstrapEditor
 			Clipboard, Enter, Typing, Paragraph, EasyImage, Image, ImageUpload, CloudServices,
 			BoldEditing, ItalicEditing, UnderlineEditing, HeadingEditing, UndoEditing
 		],
-		cloudServices: CS_CONFIG
+		cloudServices: CS_CONFIG,
+		ui: {
+			viewportOffset: {
+				top: window.getViewportTopOffsetConfig()
+			}
+		}
```

Comment:
This will have no effect because in this (Bootstrap) editor there is no core editor UI used that could understand this configuration.

Reply:
Wasn't really able to verify this kind of thing, so I just added it everywhere :) Are there more like this one that I should remove the snippet from?